gethwrappers/generation/zksync/wrap.go

```diff
@@ -14,6 +14,11 @@ func main() {
 	contractName := os.Args[2]
 	packageName := os.Args[3]
 
+	className := contractName
+	if len(os.Args) > 4 {
```

Comment:
Why do we need this?

Reply:
We are using the contract name as the class name, similar to the EVM generation. In this case we want to put this code inside the `WETH9` class but the contract was renamed

Reply:
Why do you want to put the code inside the WETH9 class? My comment about renaming was also meant to ensure the offchain code is not confusing by having the same names

Reply:
For simplicity. The contract does virtually the same and it is only used for testing. The wrapper for ZK adds the deploy function to existent classes.

I can craft a new wrapper for this specific case: new package, new class name, but using the ABI of the existent `WETH9` class. That would be simple to consume from `chainlink/deployment` too. What do you think?

Reply:
If there's a single byte that's different, it's a different contract. Renaming files like this makes it harder to find the source code, as that is now names differently. I think it's better for consistency to not rename anything